docs/public_api_test.py

```diff
@@ -70,7 +70,7 @@ class PublicAPITest(absltest.TestCase):
     def test_public_symbols(self):
         """Asserts that the symbols exposed under tp.<> are exactly the
         ones we expect."""
-        symbols = {s for s in dir(tp) if not s.startswith("__")}
+        symbols = {s for s in dir(tp) if not s.startswith("__") and s != "api"}
```

Comment:
`del api` at the end of the root `__init__` is the correct fix for this I believe

Reply:
Good point.
*Removed condition and added "del api".